landoui/revisions.py

```diff
@@ -201,6 +201,20 @@ def revision(revision_id):
                 submitted_rev_url = rev["url"]
                 break
 
+    # Current implementation requires that all commits have the flags appended.
+    # This may change in the future. What we do here is:
+    # - if all commits have the flag, then disable the checkbox
+    # - if any commits do not have the flag, then enable the checkbox
+
+    if target_repo:
+        existing_flags = {f[0]: None for f in target_repo["commit_flags"]}
```

Comment:
Nit: expanding the tuple `f` into it's parts would make this slightly easier to parse. I _think_ the change below would be correct.

```python
{flag: None for flag, _help in target_repo["commit_flags"]}
```

Reply:
I agree that naming the variable (i.e. `flag`) adds value and readability, but I think that not assigning an unneeded variable also adds some value. The name of the list (`existing_flags`) implies that the list contains flags, so I think it's a little cleaner to keep things here as they are. Will not choose this nit, as you've indicated :)